setup.py

```diff
@@ -46,7 +48,7 @@
     package_dir={"": "src"},
     include_package_data=True,
     zip_safe=False,
-    python_requires="==2.7",
+    python_requires="~=3.7, ~=3.8",
```

Comment:
Simpler and the same would be only this: `~=3.7`.

Reply:
I had understood from the documentation that `~=3.7` was for the `3.7` series only. He wouldn't accept `3.8`. See:

https://www.python.org/dev/peps/pep-0440/#compatible-release

Reply:
The example they give, is that `~= 2.2` is the same as `>= 2.2, == 2.*`, so at least 2.2, but any higher version in the 2.x range is fine.
In our case, `~=3.7` would mean 3.7*, 3.8*, 3.9*, but not 4.*.
We could also simply say `>=3.7`: everyone understands that.

Better would actually be 3.6 as minimum really, as Plone 5.2 itself supports it. Even if we don't test on 3.6, it is nicer to not make it impossible for users on 3.6 to install it, which is the case when you use pip. (Buildout does not care about `python_requires`, at least not Buildout 2.)

And when you restore Python 2 support, it should be this ugly beast:

    python_requires='>=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*,!=3.4.*!=3.5.*',

Reply:
> And when you restore Python 2 support, it should be this ugly beast:
>  python_requires='>=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*,!=3.4.*!=3.5.*',

Thanks! I'll still put a <3.9 at the end to prevent Python 3.9 and 3.10.

Reply:
I would not add an upper bound (`<3.9`) unless you know it *really* will not work. With this restriction in place, if someone wants to try this package out on Plone 6 with Python 3.9, installation will fail. I would not want that. There is probably a good chance that it would work without changes.

Reply:
Ok. I removed it.